deluge/plugins/AutoAdd/deluge/plugins/autoadd/core.py

```diff
@@ -252,12 +252,13 @@ def update_watchdir(self, watchdir_id):
                 # Skip directories
                 continue
             else:
-                ext = os.path.splitext(filename)[1]
+                ext = os.path.splitext(filename)[1].lower()
```

Comment:
To be picky, I wouldn't modify `ext` here, as it's not strictly the filename extension. It is better to do it temporarily in the test below `if exe.lower() ==`.